templates/web/bromley/reports/_list-entry.html

```diff
@@ -0,0 +1,6 @@
+<li class="[% c.cobrand.pin_colour(problem) %]">
```

Comment:
I've changed this template so that it contains more of the information again - you're right, the new filtering work seems to have dropped a bunch of stuff that there's no real need to. I think this is because it came from collideoscope originally, where a lot of the info isn't available or appropriate.

I looked into fixing this on a bigger scale, but things get a bit confusing between the _list-entry and _item templates, and the different types of list they can be in (.report-list, .report-list-a, .issue-list-a and so on). I'll ticket tidying it up better (perhaps along with including these filters as the default?).